README.md

```diff
@@ -122,6 +122,18 @@ testprofiler {
              * A method to add additional suffixes
              */
             addTestClassNameSuffix 'SomeOtherSuffix'
+            
+            /**
+             * List of regexps related to FQN of class that if matched will NOT break the test
+             */
+            List<String> testClassRegexpsToIgnore = []
+    
+            /**
+             * A method to add regexps of classes to ignore
+             */
+            void addTestClassRegexpToIgnore(String... testClassRegexpToIgnore) {
```

Comment:
I'm defining all possibilities in the readme so that people don't have to go to the code to see the options